cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KRaftMetadataManagerTest.java

```diff
@@ -46,6 +44,8 @@
 public class KRaftMetadataManagerTest {
     private static Vertx vertx;
 
+    private static final TlsPemIdentity DUMMY_IDENTITY = new TlsPemIdentity(null, null);
```

Comment:
Should this be in some shared class instead of being created in every class? ALso, if this should be here, it should be above the `vertx` field as it is `final static`.